1-js/02-first-steps/09-comparison/1-comparison-questions/task.md

```diff
@@ -2,17 +2,16 @@ importance: 5
 
 ---
 
-# Comparisons
+# Համեմատություններ
 
-What will be the result for these expressions?
+Ի՞նչ են հետևյալ արտահայտությունների արժեքները:
 
 ```js no-beautify
 5 > 4
 "apple" > "pineapple"
 "2" > "12"
 undefined == null
 undefined === null
-null == "\n0\n"
-null === +"\n0\n"
-```
-
+null == "0"
+null === "0"
```

Comment:
@ArMinasyan please don't change those values.

Reply:
Which part?

Reply:
null == +"0" this?

Reply:
Here are the original values:
```
null == "\n0\n"
null === +"\n0\n"
```
Possibly your editor is messing up with those values.